### ISSUE TITLE
Puppet Stayman

### ISSUE THREAD

**Issue by ThorvaldAagaard** [EdwardPiwowar/BBA#234]
I can see Puppet Stayman is greyed out. When you add it will you also add a version after 2N? 
https://mjbridge.info/responder2NT/page77.html
https://chrisryall.net/bridge/debates/puppetry.htm
https://bridgewinners.com/article/view/what-type-of-stayman-after-a-2nt-opening/

**Comment by EdwardPiwowar** [comment]
I'm already working on it, will Puppet only be enough for 2NT?

**Comment by ThorvaldAagaard** [comment]
For BEN yes, but I am not sure, what other robots are playing. Jack has puppet after 1N as default

**Comment by Muppet1** [comment]
Jack plays 1NT 3 clubs as puppet.  But don't worry about it go for stuff ben wants to play.

**Comment by ThorvaldAagaard** [comment]
Yes, but Ian is training a BEN based on Jack data, so he is interested in getting that to work :-)

**Comment by Muppet1** [comment]
sorry in case it was confusing muppet= Ian.   Net is in fact running now with info file.   I'll email the results later.
